pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -276,7 +276,19 @@ func (ds *dockerService) StopPodSandbox(ctx context.Context, r *runtimeapi.StopP
 		return resp, nil
 	}
 
-	// TODO: Stop all running containers in the sandbox.
+	go func() {
+		containers, err := ds.client.ListContainers(dockertypes.ContainerListOptions{All: true})
```

Comment:
I'm afraid this is not right. Listing all containers would also include containers in other pod sandboxes.

Reply:
Thanks. I have updated the code , set up filter before called ``ListContainers()`` func.
I think it should work, but I have no idea why unit test failed. 
```
opts.Filters = dockerfilters.NewArgs()
f := newDockerFilter(&opts.Filters)
f.AddLabel(sandboxIDLabelKey, podSandboxID)

containers, err := ds.client.ListContainers(opts)
```

Reply:
You can check the details for each failed cases. If you think they shouldn't happen, comment with `/retest` would rerun them.

Reply:
Thanks. After retest all unit tests passed.